src/Transition/InputRulesFiles/Rules8-7-0-to-8-8-0.md

```diff
@@ -8,17 +8,22 @@ This was previously an Excel workbook that made for very difficult version contr
 
 Insert new blank field 5 (A5) (for System Inlet Air Node Name). 
 Shift all later fields down by one.
-=======
 
 # Object Change: `Table:TwoIndependentVariables`
 
 Insert new blank field 14 (A7) (for External File Name). 
 Shift all later fields down by one.
-=======
 
 # Object Change: `Output:Surfaces:List`
 
 The only change is for field F1, which is A1.  Logic to apply:
 
 if key = 'DecayCurvesfromZoneComponentLoads', change to = 'DecayCurvesFromComponentLoadsSummary'
 
+# Object Change: `UnitarySystemPerformance:Multispeed`
+
+Fields 1-4 remain the same.  
+After F4, insert one new blank field that represents the No Load Supply Air Flow Rate Ratio. 
+Shift all later fields down by one.
```

Comment:
OK, there is something sketchy going on.  I went to add this transition rule to the Fortran source, and found that it wouldn't compile, because there was already a rule in there for this object.  I thought, OK, this is probably just one of those uncommon situations where a second transition rule is applied, and we need to carefully merge them together.  But then I noticed they were the exact same rule.  Weird.  I went back and found I had added the transition rule in as part of the many transition rules for #6158 [here](https://github.com/NREL/EnergyPlus/pull/6158/files#diff-1a3d976fbe90f0b4b2131917bff01b5fR369).  I added it there because it was one of the requested additions in the Rules file [there](https://github.com/NREL/EnergyPlus/pull/6158/files#diff-84d66175020363cd70e1e9161a145675R23).  But I can't figure out why that was in that pull request, when it's really a part of this PR.

Either way, the "Rules" file and the Fortran transition source both have the no-load field in it.  And now with this PR, the IDD and code finally have the changes in them.  It seems like everything will be back to compatible once this PR goes in.  Can you confirm?